data/core.yaml

```
@@ -774,9 +774,15 @@ en:
     title: Background
     description: Background Settings
     key: B
-    backgrounds: Backgrounds
+    backgrounds:
+      photo: Aerial and Satellite Imagery
+      map: Maps and Terrain Cartography
+      osm: Maps based on OpenStreetMap
```

A bit more concise (and in title case for consistency):

```suggestion
      osm: OpenStreetMap-Based Maps
```